src/pages/index.js

```diff
@@ -65,3 +65,57 @@ profileFormValidator.enableValidation();
 
 const cardsFormValidator = new FormValidator(config, cardsEditForm);
 cardsFormValidator.enableValidation();
+
+//
+
+export const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "f01264e8-2101-4ab7-b120-b09b55e63681",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .returnUserInfoAndCards()
+  .then((res) => {
+    profileEdited.setUserInfo(res[0]);
+
+    propic.src = res[0].avatar;
+
+    const newCardsSection = new Section(
+      {
+        data: res[1],
+        renderer: (item) => {
+          const cardElement = createCard(item);
+          cardList.append(cardElement);
+        },
+      },
+      cardList
+    );
+    newCardsSection.renderItems();
+  })
+  .catch((err) => console.error(err));
+
+const propicPopup = new PopupWithForm(propicModal, {
```

Comment:
`propicPopup`

Please, name it `avatarPopup` to make it clear instead of `propic`